cmd/ddev/cmd/debug-fixcommands.go

```diff
@@ -2,31 +2,19 @@ package cmd
 
 import (
 	"github.com/ddev/ddev/pkg/ddevapp"
-	"github.com/ddev/ddev/pkg/globalconfig"
 	"github.com/ddev/ddev/pkg/util"
 	"github.com/spf13/cobra"
 )
 
-// DebugFixCommandsCmd fixes up custom/shell commands without having to do a ddev start
+// DebugFixCommandsCmd fixes up global container commands without having to do a ddev start
 var DebugFixCommandsCmd = &cobra.Command{
 	Use:   "fix-commands",
-	Short: "Fix up custom/shell commands without running ddev start",
+	Short: "Fix up global container commands without running ddev start",
 	Run: func(_ *cobra.Command, _ []string) {
-		app, err := ddevapp.GetActiveApp("")
-		if err != nil {
-			util.Failed("Can't find active project: %v", err)
-		}
-		err = ddevapp.PopulateCustomCommandFiles(app)
+		err := ddevapp.PopulateGlobalCustomCommandFiles()
 		if err != nil {
 			util.Warning("Failed to populate custom command files: %v", err)
 		}
-		// If no-bind-mounts we have to do a start to push the commands back in there again.
-		if globalconfig.DdevGlobalConfig.NoBindMounts {
-			err = app.Start()
-			if err != nil {
-				util.Failed("Failed to restart with NoBindMounts set: %v", err)
-			}
-		}
```

Comment:
I **think** it's okay to remove this, but I'd like another set of eyes over it since I'm not quite sure.

Reply:
I do not believe this should be removed, but haven't studied your situation carefully.

You can easily experiment with `--no-bind-mounts` and test it. It's very rarely used, is intended for remote docker server. It was actually created when Colima didn't have bind mounts. But just turn it on and see if commands work.

Reply:
Just tried this, and it seems it's not needed? I ran `ddev config global --no-bind-mounts=true`, then `ddev start` to get my project up.
Then modified a global web command and ran `ddev debug fix-commands` and the change was reflected in `/mnt/ddev-global-cache/global-commands/web/`.